manager-bundle/composer.json

```diff
@@ -55,6 +55,7 @@
         "symfony/web-profiler-bundle": "^4.4 || ^5.2",
         "symfony/yaml": "^4.4 || ^5.2",
         "toflar/psr6-symfony-http-cache-store": "^2.1 || ^3.0",
+        "twig/extra-bundle": "^2.11.3",
```

Comment:
Isn't it a little odd that the core bundle requires `twig/twig` but the manager bundle requires `twig/extra-bundle`?

Reply:
My reasoning was: in a non-managed edition you would install `twig/extra-bundle` and register it in the kernel yourself because you need it for your app. Twig however is needed by the core as well, so it must be required there.